org.mozilla.FirefoxNightly/org.mozilla.FirefoxNightly.json

```diff
@@ -5,6 +5,16 @@
     "sdk": "org.gnome.Sdk",
     "command": "firefox",
     /*"tags": [""],*/
+    "add-extensions": {
+        "org.mozilla.FirefoxNightly.flashplayer": {
+	    "directory": "lib/mozilla/plugins",
+	    "add-ld-path": "lib",
+	    "merge-dirs": "lib/mozilla/plugins",
+	    "subdirectories": true,
+	    "no-autodownload": true,
```

Comment:
If we have just a specific extension point for the Flash plugin, I would be tempted to set autodownload to true, in terms of making stuff just work.